com.liferay.blade.upgrade.liferay70/src/com/liferay/blade/upgrade/liferay70/apichanges/MovedClassesImport.java

```diff
@@ -0,0 +1,1449 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.liferay.blade.upgrade.liferay70.apichanges;
+
+import com.liferay.blade.api.AutoMigrator;
+import com.liferay.blade.api.FileMigrator;
+import com.liferay.blade.upgrade.liferay70.ImportStatementMigrator;
+
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+	property = {
+		"file.extensions=java",
+		"problem.title=Moved Classes Import Error",
```

Comment:
Is there not a breaking changes item we can link to?  could we add a section to breaking changes so this Migrator could link to some documentation?